spirv_msl.hpp

```diff
@@ -870,6 +873,8 @@ class CompilerMSL : public CompilerGLSL
 	// Must be ordered since array is in a specific order.
 	std::map<SetBindingPair, std::pair<uint32_t, uint32_t>> buffers_requiring_dynamic_offset;
 
+	std::unordered_set<uint32_t> disabled_frag_outputs;
```

Comment:
These variables need to be emitted in order to make test suite stable, so I suggest a vector here since there are generally few elements anyways.